layouts/error.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <v-row justify="center" class="mt-16 mb-4">
+      <v-icon :color="icon.color" size="72">{{ icon.icon }}</v-icon>
+    </v-row>
+    <v-row justify="center">
+      <h1 class="text-h4">{{ error.message }}</h1>
```

Comment:
```suggestion
      <h1 class="text-h4 text-center">{{ error.message }}</h1>
```

just centers text on small screens.

Reply:
Nice catch, done